### PR TITLE
[MI-2925]: Open list on Input focus

### PR DESCRIPTION
**Pull request by SaurabhSharma-884**

### CODE REVIEW

File: src/components/Input/Input.component.tsx [hunk 1]
```diff
@@ -86,9 +87,12 @@ export const Input = forwardRef<HTMLInputElement, InputProps>((props, ref) => {
...
 					placeholder={inputLabel}
-					onFocus={(event: React.ChangeEvent<HTMLInputElement>) =>
-						togglePlaceholderValue(event, 'focus')
-					}
+					onFocus={(event: React.ChangeEvent<HTMLInputElement>) => {
+						if(onInputFocus){
+							onInputFocus();
+						}
+						togglePlaceholderValue(event, 'focus');
+					}}
```
Comment (sooraj-shukla) [inline src/components/Input/Input.component.tsx:95]: Please fix the discussed issue in bootstrap migration PR for input component.

File: src/components/Input/Input.d.ts [hunk 2]
```diff
@@ -100,4 +100,9 @@ export interface InputProps {
 	 * Handler that's called when there is a change in the value of the input field
 	 */
     onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+
+    /**
+	 * Handler that's called when input field gains focus
```
Comment (sooraj-shukla) [inline src/components/Input/Input.d.ts:105]:
```suggestion
	 * Handler for handling input focus event
```

**Review by sooraj-shukla — CHANGES_REQUESTED** [review]

File: src/components/Input/Input.component.tsx (hunk 1 above)
Comment (SaurabhSharma-884) [reply src/components/Input/Input.component.tsx:95]: Can fix discussed changes in https://github.com/Brightscout/mm-ui-library/pull/72

File: src/components/Input/Input.d.ts (hunk 2 above)
Comment (SaurabhSharma-884) [reply src/components/Input/Input.d.ts]: Fixed

**Review by sooraj-shukla — APPROVED** [review]

**Review by avas27JTG — APPROVED** [review]

**Pull request merged**